### ISSUE TITLE
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

### ISSUE THREAD

**Issue by yangyang1-0** [datawhalechina/self-llm#267]
使用 `Qwen2.5-7B-Instruct FastApi 部署调用` 文档部署 `Qwen2.5-32B-Instruct` 模型，在最后的服务启动后，尝试使用 curl 测试推理，但是发送curl 命令后，服务打印 The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results. 后边没响应了。
![image](https://github.com/user-attachments/assets/57ffcdec-6223-4fe6-8c16-63332237887b)

**Comment by weijia24K** [comment]
兄弟，解决了吗？我也卡在这儿
